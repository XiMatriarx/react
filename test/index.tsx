import '../src/index.tsx'

describe('Index', () => {
  test('Snapshot', async () => {
    await new Promise((resolve) => setTimeout(resolve, 200))
    expect(document.getElementById('root')).toMatchSnapshot()
  }, 2000)
})
