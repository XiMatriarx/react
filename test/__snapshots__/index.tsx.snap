// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Snapshot 1`] = `
<div
  id="root"
>
  <div
    id="app"
  />
</div>
`;
